app/api/routers/breeze_buddy/numbers/handlers.py

```
@@ -159,6 +162,66 @@ async def get_number_handler(number_id: str, current_user: UserInfo) -> Outbound
+            outbound_number_id=number_id,
+            ivr_config=ivr_config_dict,
+        )
+
+        if outbound_number:
+            logger.info(f"Outbound number {number_id} IVR config updated successfully")
+            return outbound_number
+        else:
+            raise HTTPException(
+                status_code=status.HTTP_400_BAD_REQUEST,
+                detail="Failed to update outbound number IVR config",
+            )
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)